gokart/task.py

```diff
@@ -112,7 +112,7 @@ def make_target(self, relative_file_path: str, use_unique_id: bool = True, proce
         unique_id = self.make_unique_id() if use_unique_id else None
         return gokart.target.make_target(file_path=file_path, unique_id=unique_id, processor=processor)
 
-    def make_large_data_frame_target(self, relative_file_path: str, use_unique_id: bool = True, max_byte=int(2**26)) -> TargetOnKart:
+    def make_large_data_frame_target(self, relative_file_path: str, use_unique_id: bool = True, max_byte=int(2 ** 26)) -> TargetOnKart:
```

Comment:
[nits]

> int(2 ** 26)

is this `int()` needed? It is OK when to show your intent explicitly.